{{ cookiecutter.package_name }}/setup.py

```diff
@@ -66,6 +60,13 @@
     __import__(PACKAGENAME)
     package = sys.modules[PACKAGENAME]
     LONG_DESCRIPTION = package.__doc__
+    __minimum_python_version__ = package.__minimum_python_version__
```

Comment:
I think this should also be coming from cookiecutter as  ``__init__.py`` is not yet run at the time running setup.py

Reply:
Isn't that the point of this block? To allow you to import the package?

Reply:
yet somehow that didn't work in the core package, see its setup.